deploy/charts/cert-manager/templates/webhook-mutating-webhook.yaml

```diff
@@ -18,14 +18,13 @@ webhooks:
     rules:
       - apiGroups:
           - "cert-manager.io"
-          - "acme.cert-manager.io"
         apiVersions:
           - "v1"
         operations:
           - CREATE
           - UPDATE
         resources:
-          - "*/*"
+          - "certificaterequests"
```

Comment:
One thing to note: this does mean we won't mutate any *subresource* requests for certificaterequests anymore. I think this is totally fine - the only time we need to inspect requests to `certificaterequests/status` is in the `approval` admission plugin: https://github.com/cert-manager/cert-manager/blob/master/internal/plugin/admission/certificaterequest/approval/certificaterequest_approval.go

This is a *validating* plugin only however, not mutating, so this is fine 👍